docs/modules/operation/pages/deep-dive/search.adoc

```diff
@@ -1,31 +1,29 @@
 
 = Search
 
-You can search in the {page-component-title} web UI using either the search bar or the Search view.
+You can search your {page-component-title} data in the web UI using either the search bar at the top of each page or the Search view.
```

Comment:
The search also returns areas of the UI, not just your data. You can type "manage" and it will show a list of tasks and when you click it takes you to that area of the UI.

Reply:
```suggestion
You can search in the {page-component-title} web UI using either the search bar at the top of each page or the Search view.
```